libvast_test/vast/test/test.hpp

```diff
@@ -13,16 +13,39 @@
 #endif
 
 #include "vast/span.hpp"
+#include "vast/test/type_ids.hpp"
 
+#include <caf/allowed_unsafe_message_type.hpp>
+#include <caf/detail/stringification_inspector.hpp>
+#include <caf/inspector_access.hpp>
 #include <caf/test/unit_test.hpp>
+#include <fmt/format.h>
 
 #include <optional>
 #include <set>
 #include <string>
+#include <string_view>
 
 // Work around missing namespace qualification in CAF header.
 using ::caf::term;
 
+namespace caf {
+
+template <>
+struct inspector_access<std::string_view> {
+  static auto apply(detail::stringification_inspector& f, std::string_view& x) {
+    auto str = std::string{x};
+    return f.apply(str);
+  }
+
+  static bool
+  save_field(detail::stringification_inspector&, string_view, auto&, auto&) {
+    return true;
+  }
+};
+
+} // namespace caf
```

Comment:
This looks _really_ dangerous, even just for unit tests. What do we need this for?

Reply:
It is related to data_view comparison i believe. I should be able to remove it